packages/math-input/src/components/keypad/keypad-button.tsx

```diff
@@ -0,0 +1,128 @@
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import {KeyConfig, ClickKeyCallback} from "../../types";
+
+import ButtonAsset from "./button-assets";
+
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+
+export type KeypadButtonProps = {
+    // 0 indexed x,y position in keypad CSS grid
+    coord: readonly [column: number, row: number];
```

Comment:
I can change this to `coord: {column: number, row: number};` if we really want, but I think for an x/y pair `[0, 0]` makes sense.

Reply:
Named tuples members feel "ok".  It get's obscured in the callsites as you don't have the tuple names.  We do have alot of places in Perseus where we use tuples for coordinates, sizes, and ranges... I think I'd prefer the verbosity of objects if I had my way. :)